tests/run/pyclass_annotations_pep526.py

```diff
@@ -0,0 +1,45 @@
+# cython: language_level=3
+# mode: run
+# tag: pure3.7, pep526, pep484
+
+from __future__ import annotations
+
+
+from typing import ClassVar
```

Comment:
A `try-except` would make this work in Py<3.5 as well.

Reply:
Wasn't sure how we felt about that - is the eventual goal to drop support of Py<3.5 alongside dropping Py2?